.github/workflows/sanity.yml

```diff
@@ -0,0 +1,76 @@
+name: Sanity
+
+on:
+  pull_request:
+    paths:
+      - "data/**"
+      - "python/**"
+
+jobs:
+  main:
+    runs-on: "ubuntu-latest"
+    container:
+      image: "quay.io/centos/centos:stream9"
+
+    steps:
+      - name: "Checkout the repository"
+        uses: actions/checkout@v4
+
+      - name: "Install as a Python package"
+        run: |
+          dnf --setopt install_weak_deps=False install -y python3-pip gpg make git-core
+          python3 -m pip install --upgrade pip
+          python3 -m pip install .
+
+      - name: "Install insights-client"
+        run: |
+          curl https://copr.fedorainfracloud.org/coprs/g/yggdrasil/latest/repo/centos-stream-9/group_yggdrasil-latest-centos-stream-9.repo \
+            -o /etc/yum.repos.d/copr.repo
+          dnf install -y insights-client
+
+      # We need to replace the GPG keys included in insights-core with our own keys.
+      # For that, we need to:
+      # - download Insights Core
+      # - generate GPG keys
+      # - replace the public key in Core
+      # - sign list of revoked plays and replace it in Core
+      # - sign own playbook
+      - name: "Prepare environment"
+        run: |
+          git clone https://github.com/RedHatInsights/insights-core.git --depth 1
```

Comment:
simply use a github action for this:
```yaml
- uses: actions/checkout@v4
  with:
    repository: RedHatInsights/insights-core
    path: insights-core
```

Reply:
I'd say this will slightly increase maintenance as next versions of the checkout action come out, but sure.